utils/testutils/testutils.go

```diff
@@ -27,3 +31,37 @@ func ReadTestFile(name string) ([]byte, error) {
 		return data.Bytes(), nil
 	}
 }
+
+// GetInterface returns the best match of an interface that might be listening on a given port.
+// This is helpful when a test is being run in a CI environment under docker.
+func GetInterface(port int) string {
+	dial := func(iface string, port int) bool {
+		c, err := net.Dial("tcp", iface+":"+strconv.Itoa(port))
+		if err != nil {
+			return false
+		}
+		c.Close()
+		return true
+	}
+	// First, we check dockerhost
+	iface := "dockerhost"
```

Comment:
Constant?

Reply:
I am actually changing the variable later. But 0/5 on it frankly. Let me know.

Reply:
Oh didn't see it was the same variable. Nevermind then